src/components/MAPI/releases/ClusterDetailWidgetRelease.tsx

```diff
@@ -159,10 +159,19 @@ const ClusterDetailWidgetRelease: React.FC<
   }, [isAdmin, isImpersonatingNonAdmin, provider, releaseList, releaseVersion]);
 
   const nextVersion = useMemo(() => {
-    return supportedUpgradeVersions.find(
+    if (!releaseList || !releaseVersion) return undefined;
+
+    const activeVersions = getSupportedUpgradeVersions(
+      releaseVersion,
+      provider,
+      false,
```

Comment:
Maybe we could add a comment here explaining that we set admin to `false` because we don't want to prompt anyone, including admins "Upgrade available" for inactive versions.
What do you think?